cmd/main.go

```diff
@@ -190,15 +190,25 @@ func main() {
 			log.WithError(cerr).Fatalf("Failed to create kubernetes cluster config")
 		}
 		k8sClient := kubernetes.NewForConfigOrDie(cfg)
+
+		err = automigrationWithLeader(k8sClient, db, log)
+		if err != nil {
+			log.WithError(err).Fatal("Failed auto migration process")
+		}
+
 		lead = leader.NewElector(k8sClient, Options.LeaderConfig, "assisted-service-leader-election-helper",
 			log.WithField("pkg", "monitor-runner"))
+
 		err = lead.StartLeaderElection(context.Background())
 		if err != nil {
 			log.WithError(cerr).Fatalf("Failed to start leader")
 		}
 
 	case "onprem":
 
+		if err = db.AutoMigrate(&models.Host{}, &common.Cluster{}, &events.Event{}).Error; err != nil {
+			log.Fatal("failed to auto migrate, ", err)
+		}
```

Comment:
Add 
log.Infof("Finish automigration")

Reply:
i have in the function. can move it here though